firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java

```diff
@@ -131,7 +131,28 @@ public Builder setPersistenceEnabled(boolean value) {
      * @return A settings object on which the return type for timestamp fields is configured as
      *     specified by the given {@code value}.
      */
+    /**
```

Comment:
Uh... This seems like you added the new but didn't remove the old?

Reply:
Gah, sorry. I should have sanity-checked the diff before sending for review but I was too busy not running the unit tests.